src/library/basesqltablemodel.cpp

```diff
@@ -20,7 +20,9 @@
 #include "util/dnd.h"
 #include "util/assert.h"
 
-const bool sDebug = false;
+static const bool sDebug = false;
+static const int kIdColumn = 0; // Must not be changed
```

Comment:
(what I mean is -- what's stopping someone from calling setTable with item 0 not equal to the index column?)